applications/solvers/additiveFoam/setDeltaT.H

```diff
@@ -4,20 +4,25 @@ if (adjustTimeStep)
     scalar maxAlphaCo =
         runTime.controlDict().lookupOrDefault<scalar>("maxAlphaCo", 0.1);
 
-    const scalar dtBydT = mag(runTime.deltaT() / (Tliq - Tsol)).value();
+    alphaCoNum = gMax(mag(T - T.oldTime())()) / (Tliq - Tsol).value();
 
-    alphaCoNum = gMax(mag(fvc::ddt(T))())*dtBydT;
+    Info << "alphaCoNum: " << alphaCoNum << endl;
 
-    Info << "Courant Number thermo: " << alphaCoNum << endl;
+    scalar maxFo = runTime.controlDict().lookupOrDefault<scalar>("maxFo", 1);
 
-    FoNum = gMin(Foam::pow(mesh.V().field(), 2.0/3.0) / (kappa/rho/Cp)());
+    const scalar diffusionTime
```

Comment:
In the existing OF solid solver, a closely related quantity is calculated as:

```
    const volScalarField::Internal DiNumvf
    (
        fvc::surfaceSum
        (
            mesh.magSf()
           *fvc::interpolate(kappa)
           *mesh.surfaceInterpolation::deltaCoeffs()
        )()()
       /(mesh.V()*thermo.rho()()*thermo.Cp()())
       *runTime.deltaT()
    );
```

Not sure if it really matters which way it is done but I wanted to bring it to your attention.

Reply:
Good find. This increases the explicit stability limit by about 20%. Added in [be374d3](https://github.com/ORNL/AdditiveFOAM/pull/16/commits/be374d33f2660d38271ac4de8dfdf082505c545e)

Reply:
The explicit limit using this scheme was found to be maxDi = 1